src/xdocs/idea.xml

```diff
@@ -77,18 +77,28 @@
       <p>
         Checkstyle has its own very strict set of inspections. To import and enable them go to
         Settings -> Editor -> Inspections -> Manage -> Import... and locate file
-        <code>config/intellij-idea-inspections.xml</code>.<br/><br/>
+        <code>config/intellij-idea-inspections.xml</code>.
+        <br/>
+        <br/>
         <img alt="Inspections in IntelliJ IDEA" src="images/gui_screenshot_inspections_idea.png"/>
-        <br/><br/>
-        Not all files in repository should be analyzed. For example test input files contain
-        numerous violations on purpose. Our shared inspection scope can be used to exclude such
-        files.<br/>
-        To add custom scope copy file <code>config/intellij-idea-inspection-scope.xml</code>
+        <br/>
+        <br/>
+        ATTENTION: Not all files in repository should be analyzed.
+        For example test input files contain
```

Comment:
Formatting of this line looks off. I thought you split it by periods, but this whole sentence could fit on 1 line, up to the period.
Can you move `numerous violations on purpose.` to this line.
Make `Our shared` all one 1 line too.

Reply:
done.